frontend/src/pages/SingleInstancePage.tsx

```diff
@@ -30,67 +30,74 @@ export default function SingleInstancesPage() {
     { label: "Jobs", content: <JobsTable jobs={jobs} /> },
   ];
   return (
-    <div className="flex h-full w-full flex-col gap-3">
+    <div className="flex h-full w-full flex-col">
       <ResizableContainer direction="vertical">
         <div className="flex h-full">
           <ResizableContainer direction="horizontal">
-            <div className="w-full pr-3">
-              <Table
-                orientation="vertical"
-                header={[
-                  "Instance Key",
-                  "BPMN Process ID",
-                  "Version",
-                  "Process Key",
-                  "Status",
-                  "Start Time",
-                  "End Time",
-                ]}
-                content={
-                  instance
-                    ? [
-                        [
-                          instanceKey,
-                          bpmnProcessId,
-                          version,
-                          <NavLink to={`/processes/${processKey}`}>
-                            <Button variant="secondary">{processKey}</Button>
-                          </NavLink>,
-                          status,
-                          startTime,
-                          endTime,
-                        ],
-                      ]
-                    : []
-                }
-              />
+            <div className="w-full overflow-hidden">
+              <div className="min-w-[350px] pr-3">
+                <Table
+                  orientation="vertical"
+                  header={[
+                    "Instance Key",
+                    "BPMN Process ID",
+                    "Version",
+                    "Process Key",
+                    "Status",
+                    "Start Time",
+                    "End Time",
+                  ]}
+                  content={
+                    instance
+                      ? [
+                          [
+                            instanceKey,
+                            bpmnProcessId,
+                            version,
+                            <NavLink to={`/processes/${processKey}`}>
+                              <Button variant="secondary">{processKey}</Button>
+                            </NavLink>,
+                            status,
+                            startTime,
+                            endTime,
+                          ],
+                        ]
+                      : []
+                  }
+                />
+              </div>
             </div>
```

Comment:
Why is this a Table inside a div inside another div? can it be at least merged into one div instead of 2?